scripts/superdesk/editor2/editor.ctrl.js

```diff
@@ -366,7 +367,23 @@ function SdTextEditorController(_, EMBED_PROVIDERS, $timeout, $element, editor,
                 }
                 return hash;
             }
-            return String(Math.abs(hashCode(block.body))) + String(vm.getBlockPosition(block));
+
+            var blockId = String(Math.abs(hashCode(block.body)));
+
+            if (vm.blockIds.indexOf(blockId) !== -1) {
```

Comment:
An inline comment describing what happens here would help out reviewers and maintainers.